### PR TITLE
Quick update of existing documentation

### PR DESCRIPTION
**Pull request by ptomato**
https://phabricator.endlessm.com/T14835

### CODE REVIEW

File: docs/flatpak-bundle.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Anatomy of a Knowledge App Flatpak Bundle
+
+Let's look at a simple knowledge app. Knowledge apps are shipped as part of a bundle, and
+you can fetch a bundle from our Flatpak repository here:
+`flatpak install com.endlessm.astronomy.es eos3`. I chose com.endlessm.astronomy.es for its
```
Comment (magcius) [inline docs/flatpak-bundle.md]: Should we leave some comment about inspecting the resulting flatpak in /var/lib/flatpak?

File: docs/domain-versions.md [hunk 2]
```diff
@@ -2,14 +2,15 @@
 
 The convention used by this document is to explain paths in context of domain
 "example-en", and with files starting from the bundle root. So the file EKN_VERSION would
-be listed as `/share/ekn/data/example-en/EKN_VERSION`.
+be listed as `/share/ekn/data/com.endlessm.example.en/EKN_VERSION`.
 
 Since knowledge-lib is part of the core OS, and is updated separately from application
```
Comment (magcius) [inline docs/domain-versions.md]: This is also out of date.

File: docs/domain-versions.md [hunk 3]
```diff
@@ -2,14 +2,15 @@
...
 bundles, this means that knowledge-lib needs some form of backwards compatibility with
 bundles, even if we change parts of the bundles.
 
 A set of content is called a "domain". The format of content shipped in a domain is
-specified with a file, `/share/ekn/data/example-en/EKN_VERSION`, shipped in each bundle,
+specified with a file, `/share/ekn/data/com.endlessm.example.en/EKN_VERSION`,
+shipped in each bundle,
 which contains a number. If it's missing, it's assumed to be a version 1 domain.
 
 Domains have two major parts: 1. a Xapian database of records which is used for finding
```
Comment (magcius) [inline docs/domain-versions.md]: Should we remove text about V1/V2 now that the code doesn't support them anymore?

File: docs/flatpak-bundle.md (hunk 1 above)
Comment (ptomato) [inline docs/flatpak-bundle.md]: Fixed.

File: docs/domain-versions.md (hunk 3 above)
Comment (ptomato) [inline docs/domain-versions.md]: I ended up rewriting large portions of this altogether.

File: docs/domain.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+files to be completely self-contained pieces of searchable data. It
+is a Xapian single-file "Glass" database, containing a position table, a
+postlist table, a
+record table, etc.
+
+As an example of a metadata file, here's one:
+
+    {
+      "@type": "ekn://_vocab/ImageObject",
+      "@id": "ekn:///7ec07025139a5c22",
```
Comment (magcius) [inline docs/domain.md]: ID looks too short. Is this a legacy truncated ID?

File: docs/domain.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+On first start of
+the application, if missing, the manifest and shards will be symlinked into the user's
+home directory.
+
+At runtime, the user's computer checks for updates to each subscription and downloads new
+shards in the background.
+
+To help inspect applications and subscriptions, eos-knowledge-lib ships
+with a
+tool called `eminem`. Please run `eminem` for more details on usage.
```
Comment (magcius) [inline docs/domain.md]: Word wrap on this seems weird.

Comment (ptomato) [inline docs/domain.md]: Yeah, I was trying to leave unchanged lines unchanged in the diff (and rendered in Markdown it doesn't matter), but I changed so much it's not recognized as a rename anymore anyway, so I may as well fix up the wrapping

File: docs/domain.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+          "https://commons.wikimedia.org/wiki/File:Barbarossaleuchter.jpg",
+          "https://upload.wikimedia.org/wikipedia/commons/thumb/2/2a/Barbarossaleuchter.jpg/220px-Barbarossaleuchter.jpg"
+      ],
+      "width": 757
+      "height": 600,
+    }
+```
+
+Fields like "@type" and "@id" are part of [JSON-LD](http://json-ld.org/),
+a specification for hypermedia-style linked data.
```
Comment (magcius) [inline docs/domain.md]: You should probably mention that we don't use "real" JSON-LD, and that these schemas are roughly based on schema.org equivalents.

**Comment by magcius** [comment]
Otherwise looks good to me.

**Pull request merged**